Include limits header needed for std::numeric_limits (at least with libstdc++).
This should be reported to the developers.
--- XMPFiles/source/PluginHandler/OS_Utils_Mac.cpp.orig	2019-01-01 18:10:56.000000000 -0600
+++ XMPFiles/source/PluginHandler/OS_Utils_Mac.cpp	2019-01-10 20:10:09.000000000 -0600
@@ -15,6 +15,7 @@
 #include <CoreFoundation/CFDate.h>
 #include <CoreFoundation/CFNumber.h>
 #include <CoreFoundation/CFError.h>
+#include <limits>
 #include <string>
 
 
